cads_adaptors/constraints.py

```diff
@@ -617,6 +617,8 @@ def legacy_intersect_constraints(
             }
 
     """
+    if constraints is None or len(constraints) == 0:
```

Comment:
```suggestion
    if not constraints:
```

Reply:
I prefer the explicit, we can change if needed in the future

Reply:
Sounds good, but I think you should test both cases in the unit tests.